src/gui/optionsdialog.cpp

```diff
@@ -1445,14 +1444,14 @@ void OptionsDialog::toggleComboRatioLimitAct()
 
 void OptionsDialog::enableProxy(int index)
```

Comment:
If you don't mind, please add `const` to it.

Reply:
ok